wd_helper.au3

```diff
@@ -2922,9 +2922,12 @@ EndFunc   ;==>_WD_JsonActionKey
 ; Link ..........:
 ; Example .......: No
 ; ===============================================================================================================================
-Func _WD_GetFreePort($iMinPort, $iMaxPort)
+Func _WD_GetFreePort($iMinPort = Default, $iMaxPort = Default)
 	Local Const $sFuncName = "_WD_GetFreePort"
 	Local $iResult = 0, $iErr = $_WD_ERROR_NotFound
+
+	If $iMaxPort = Default Then $iMaxPort = ($iMinPort = Default) ? 65000 : $iMinPort
+	If $iMinPort = Default Then $iMinPort = 64000
```

Comment:
I propose to defult values
```autoit
If $iMinPort = Default Then $iMinPort = $_WD_PORT
If $iMaxPort = Default Then $iMaxPort = ($iMaxPort = Default) ? ($iMinPort + 100) : $iMaxPort
```

Reply:
@mlipok Appreciate the feedback. However, I have some concerns regarding your proposed changes --

- This function will be used in other areas (such as BiDi), so it is intended to be generic in nature and not specifically related to the port used for web driver communication

- If $_WD_Port <> 0, then that means the user has already assigned a value using _WD_Option. It makes more sense to me if these were the order of events --
```
1. Obtain a port number using _WD_GetFreePort (optional)
2. Set the port number using _WD_Option
3. Call _WD_Startup
```
- I don't see how your 2nd line modifying $iMaxPort can even function properly. Care to explain?

Reply:
Ok.
Should we also add som usage to _wd_demo.au3 ?